Sources/GoogleAI/Chat.swift

```diff
@@ -67,18 +78,33 @@ public class Chat {
 
   /// See ``sendMessageStream(_:)-4abs3``.
   @available(macOS 12.0, *)
-  public func sendMessageStream(_ parts: PartsRepresentable...)
+  public func sendMessageStream(_ parts: any ThrowingPartsRepresentable...)
     -> AsyncThrowingStream<GenerateContentResponse, Error> {
-    return sendMessageStream([ModelContent(parts: parts)])
+    return try sendMessageStream([ModelContent(parts: parts)])
   }
 
   /// Sends a message using the existing history of this chat as context. If successful, the message
   /// and response will be added to the history. If unsuccessful, history will remain unchanged.
   /// - Parameter content: The new content to send as a single chat message.
   /// - Returns: A stream containing the model's response or an error if an error occurred.
   @available(macOS 12.0, *)
-  public func sendMessageStream(_ content: [ModelContent])
+  public func sendMessageStream(_ contentClosure: @autoclosure () throws -> [ModelContent])
```

Comment:
Is there a reason the closure is named `contentClosure` here, but `content` in line 44?

Reply:
In this implementation the content is generated from the closure before starting an AsyncStream, so I needed another variable name. There's no other reason.

Reply:
I wonder if it would be better to rename `content` inside the function. I know devs don't have to type `contentClosure:` because of `_` but I wonder if it might be confusing seeing:
![Screenshot 2024-02-16 at 5 16 47 PM](https://github.com/google/generative-ai-swift/assets/3010484/69fd9272-1915-4a56-b2e2-97f24b71677c)
(no mention of autoclosure in the type in autocomplete but the name is `contentClosure`)

If leaving it as is, please update the DocC with the name `contentClosure` (line 88).

Reply:
Ah, good catch. I changed this name back and used a different name in the implementation.